indispensable/src/iosMain/kotlin/Interop.kt

```diff
@@ -0,0 +1,156 @@
+@file:OptIn(ExperimentalForeignApi::class)
```

Comment:
i don't think we should carelessly duplicate this file, it would be very easy for the two to go out of sync.

this seems like a great excuse to create the utils module we've discussed a few times now

Reply:
this is now a separate module